sys-fs/xwmfs/xwmfs-0.82.ebuild

```diff
@@ -12,7 +12,7 @@ KEYWORDS="~amd64 ~x86 ~arm"
 SLOT="0"
 
 RDEPEND="
-	sys-fs/fuse:=
+	sys-fs/fuse:0=
```

Comment:
I'm pretty sure this requires a revbump for everyone to register the change. Then, you can also update this ebuild to EAPI-7.

Also please add a sign-off as guided, info in https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin